src/schema_tools/openapi/core.cljc

```diff
@@ -353,7 +354,8 @@
                       (:title new-spec)
                       (:type new-spec))
      :in          in
-     :description ""
+     :description (or (:description spec)
```

Comment:
nitpic: `(:description spec "")`

Reply:
this handles a `:description nil` more gracefully IMO

Reply:
true that!